### PR TITLE
Closes #1195: Adds telemetry for quick action sheet

### LINKED ISSUE #1195: FNX-4863 ⁃ Telemetry for the Quick Action Bar
### Hypothesis
Our hypothesis is that the quick action bar is easy and intuitive to use: 

### What questions will you answer with this data?
* How often quick bar is being opened/closed
* Time between opened and closed (Amplitude)
* What are the most clicked actions?

### Events/Pings
* swipe up/down
* touch on each action

### Acceptance Criteria (Added by PM)
* Pings can be queried via re:dash
* We are sending telemetry events for the actions listed in the requirements
* We have documented the telemetry
* [Request Review](https://github.com/mozilla/data-review/blob/master/request.md)

### PR DESCRIPTION
**Pull request by sblatz**
### Pull Request checklist

- [x] **Quality**: This PR builds and passes detekt/ktlint checks (A pre-push hook is recommended)
- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [x] **Changelog**: This PR includes [a changelog entry](https://github.com/mozilla-mobile/fenix/blob/master/CHANGELOG.md) or does not need one
- [x] **Accessibility**: The code in this PR follows [accessibility best practices](https://github.com/mozilla-mobile/shared-docs/blob/master/android/accessibility_guide.md) or does not include any user facing features

**No tests as this is just a telemetry change**

### CODE REVIEW

**Comment by sblatz** [comment]
# Request for data collection review form

**All questions are mandatory. You must receive review from a data steward peer on your responses to these questions before shipping new data collection.**

1) **What questions will you answer with this data?**

Context Menu
- How users interact with different content (e.g. # of shares or URL copies for different type of content)

Find in page
- If the feature is usable and useful, and it'll help us figure out if there are other ways to help people find their content inline

2) **Why does Mozilla need to answer these questions?  Are there benefits for users? Do we need this information to address product or business requirements?**

* To measure how useful these features are
* Understand how people use the browser, is it a middle-man to share content, should we be making sharing easier

3) **What alternative methods did you consider to answer these questions? Why were they not sufficient?**

* N/A (These are baseline metrics)

4) **Can current instrumentation answer these questions?**

* Currently no, as these are some of the first metrics we're recording

5) **List all proposed measurements and indicate the category of data collection for each measurement, using the Firefox [data c](https://wiki.mozilla.org/Firefox/Data_Collection)[ollection ](https://wiki.mozilla.org/Firefox/Data_Collection)[categories](https://wiki.mozilla.org/Firefox/Data_Collection) on the found on the Mozilla wiki.**

* All data is Category 2.

6) **How long will this data be collected?**

Until 03/01/2020

7) **What populations will you measure?**

* All release, beta, and nightly users with telemetry enabled.

8) **Please provide a general description of how you will analyze this data.**

* Glean / Amplitude

9) **Where do you intend to share the results of your analysis?**

* Only on Glean, Amplitude and with mobile teams.

**Comment by sblatz** [comment]
@boek Made the changes you requested 😄

File: app/metrics.yaml [hunk 1]
```diff
@@ -223,6 +223,74 @@ find_in_page:
     - telemetry-client-dev@mozilla.com
     expires: "2020-03-01"
 
+quick_action_sheet:
+  opened:
+    type: event
+    description: >
+      A user opened the quick action sheet UI
+    bugs:
+    - 1036
```
Comment (boek) [inline app/metrics.yaml]: The bug field here is the issue number in Github/Bugzilla for the request to add the telemetry

Comment (sblatz) [reply app/metrics.yaml]: Doh. Totally missed this field when copy-pasting 😓 sorry!

**Pull request merged**